src/errors/Messages.js

```diff
@@ -90,6 +90,7 @@ const Messages = {
   GUILD_CHANNEL_RESOLVE: 'Could not resolve channel to a guild channel.',
   GUILD_VOICE_CHANNEL_RESOLVE: 'Could not resolve channel to a guild voice channel.',
   GUILD_CHANNEL_ORPHAN: 'Could not find a parent to this guild channel.',
+  GUILD_CHANNEL_UNOWNED: 'The fetched channel does not belong to this managers guild',
```

Comment:
```suggestion
  GUILD_CHANNEL_UNOWNED: "The fetched channel does not belong to this manager's guild.",
```

Reply:
Why change the string type?

Reply:
Because it contains a `'`. This is how eslint/prettier handle's that.

Reply:
Ah I thought it'd just be \\'